backend/utils/db.py

```diff
@@ -107,14 +107,14 @@ async def read(
             data: list; on success
             data: dict; on api call fails or errors
         """
-        request_body = {
-            "collection_name": collection_name,
-            "filter": query,
-            "object_id": resource_id,
-            "organization_id": self.organization_id,
-            "plugin_id": self.plugin_id,
-            "options": options,
-        }
+        request_body = dict(
+            collection_name=collection_name,
+            filter=query,
+            object_id=resource_id,
+            organization_id=self.organization_id,
+            plugin_id=self.plugin_id,
+            options=options,
+        )

```

Comment:
It's generally preferable to use {} notation to declare a dict. This is more "pythonic". 
Moreover, that, from a performance view, declaring a dictionary with dict() is much slower than with {}.

Reply:
Thanks.
@pauline-banye 
Could you help resolve this ?

Reply:
Hi @o4codes, I could look at this but not immediately unfortunately.

Currently out n I won't be home till much later tonight and it definitely won't get sorted before the FE meeting

Reply:
I've made the change. @pauline-banye @o4codes